app-text/llpp/llpp-30_p20190316.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI-7 should work fine here, we always try to use the latest possible EAPI available.

Reply:
I was trying to make minimal changes because I don't intend to become the maintainer. Will change.